server/store/middleware/response/response.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2016 The Chihaya Authors. All rights reserved.
+// Use of this source code is governed by the BSD 2-Clause license,
+// which can be found in the LICENSE file.
+
+package response
+
+import (
+	"github.com/chihaya/chihaya"
+	"github.com/chihaya/chihaya/server/store"
+	"github.com/chihaya/chihaya/tracker"
+)
+
+func init() {
+	tracker.RegisterAnnounceMiddleware("response", responseAnnounceClient)
+	tracker.RegisterScrapeMiddleware("reponse", responseScrapeClient)
```

Comment:
I think the name should also have the prefix "store_response" because it depends on the store server.